app/controllers/main/routes.py

```diff
@@ -81,7 +81,12 @@ def volunteerRegister():
     for the event they have clicked register for.
     """
     eventData = request.form
-    event = Event.get_by_id(eventData['id'])
+    try:
+        event = Event.get_by_id(eventData['eventId'])
+    except:
+        event = Event.get_by_id(eventData['id'])
```

Comment:
I'd just change this to whatever it needs to be, rather than accept both. If there are multiple places that hit this controller, let's just change them both to `id` or to `eventId`.